pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -159,13 +161,14 @@ public PinotHelixResourceManager(@Nonnull String zkURL, @Nonnull String helixClu
 
   public PinotHelixResourceManager(@Nonnull String zkURL, @Nonnull String helixClusterName,
       @Nonnull String controllerInstanceId, @Nonnull String dataDir) {
-    this(zkURL, helixClusterName, controllerInstanceId, dataDir, DEFAULT_EXTERNAL_VIEW_UPDATE_TIMEOUT_MILLIS, false,
-        true, true);
+    this(zkURL, helixClusterName, CommonConstants.Helix.PREFIX_OF_CONTROLLER_INSTANCE + controllerInstanceId, dataDir,
```

Comment:
controllerInstanceId should already have the prefix. Also, maybe we can remove this constructor?

Reply:
It seems it's unused. Removed it.